xml/System/Math.xml

```diff
@@ -7036,10 +7036,39 @@ The following example demonstrates how to use the <xref:System.Math.Round%28Syst
         <Parameter Name="value" Type="System.IntPtr" Index="0" FrameworkAlternate="net-6.0" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="value">A signed number.</param>
+        <summary>Returns an integer that indicates the sign of a native sized signed integer.</summary>
+        <returns>A number that indicates the sign of <paramref name="value" />, as shown in the following table.  
+  
+ <list type="table"><listheader><term> Return value  
+  
+ </term><description> Meaning  
+  
+ </description></listheader><item><term> -1  
+  
+ </term><description><paramref name="value" /> is less than zero.  
+  
+ </description></item><item><term> 0  
+  
+ </term><description><paramref name="value" /> is equal to zero.  
+  
+ </description></item><item><term> 1  
+  
+ </term><description><paramref name="value" /> is greater than zero.  
+  
+ </description></item></list></returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Examples  
+ The following example demonstrates how to use the <xref:System.Math.Sign%28System.IntPtr%29> method to determine the sign of an <xref:System.IntPtr> value and display it to the console.  
+
+ :::code language="cpp" source="~/samples/snippets/cpp/VS_Snippets_CLR/math.sign/CPP/sign.cpp" id="Snippet1":::
+ :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_CLR/math.sign/CS/sign.cs" interactive="try-dotnet" id="Snippet1":::
+ :::code language="vb" source="~/samples/snippets/visualbasic/VS_Snippets_CLR/math.sign/VB/sign.vb" id="Snippet1":::
```

Comment:
These examples do not show how to use `System.Math.Sign` with `System.IntPtr`, as the above statement implies. Either the examples need to be updated, or this `## Examples` section removed.

Same applies to the examples in the other two APIs.

Reply:
Where are the samples and what's the correct way to update them?

Reply:
Nevermind, I found them.

Reply:
In the samples folder:
https://github.com/dotnet/dotnet-api-docs/blob/main/samples/snippets/cpp/VS_Snippets_CLR/math.sign/CPP/sign.cpp